test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,111 @@
+./test_scripts/API/SetAudioStreamingIndicator/001_ATF_N_TC_SetAudioStreamingIndicator_PLAY_PAUSE_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PLAY_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_PLAY_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PLAY
+./test_scripts/API/SetAudioStreamingIndicator/003_ATF_N_TC_SetAudioStreamingIndicator_PAUSE_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/004_ATF_N_TC_SetAudioStreamingIndicator_PLAY_PAUSE_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PLAY_PAUSE
```

Comment:
@dboltovskyi, the purpose of full testing coverage is not to test the logic, but to test all(or as many as possible cases).
Rejected.